dll/win32/shell32/CDefaultContextMenu.cpp

```diff
@@ -513,9 +542,10 @@ CDefaultContextMenu::AddStaticContextMenusToMenu(
     UINT ntver = RosGetProcessEffectiveVersion();
     MENUITEMINFOW mii = { sizeof(mii) };
     UINT idResource;
-    WCHAR wszVerb[40];
+    WCHAR wszDispVerb[80]; // The limit on XP. If the friendly string is longer, it falls back to the verb key.
```

Comment:
Could you put this value as a define as well at the top? (you did it for `VERBKEY_CCHMAX` that is used once in the code

Reply:
`VERBKEY_CCHMAX` will be useful in other places, it should probably even be moved to a shared shell32 header file at some point. This display string buffer is only used here.